doc/introduction/data.rst

```diff
@@ -39,7 +31,7 @@ The :func:`~pennylane.data.load` function returns a ``list`` with the desired da
 >>> H2data = H2datasets[0]
 
 We can load datasets for multiple parameter values by providing a list of values instead of a single value.
-To load all possible values, use the special keyword "full".
+To load all possible values, use the special value :const:`~pennylane.data.FULL`.
```

Comment:
I am unaware of the discussion behind this, but wouldn't this be inconvenient to users?

Reply:
you can actually use "full" if you prefer. personally I prefer a named constant over a literal, but I can change the docs